### DISCUSSION TITLE
Getting started Project failing to display index page

### DISCUSSION THREAD

**Discussion by aadi58002** [nuxt/nuxt#16782] [Questions]
I have followed the nuxt content getting started to the letter and when i run the project. It does not show the index page instead it shows me the about page in the localhost:3000 and localhost:3000/about. Can somebody tell me the reason or if i did any step wrong. I have tried it with pnpm and npm, as well on a friend's computer.
![Screenshot_20220822_145018](https://user-images.githubusercontent.com/96418027/185887154-81d28775-45dd-42d1-864c-341d66a77774.png)

**Comment by sevillaarvin** [comment]
Can you provide a reproduction repo? In any case try the discussion at nuxt content instead https://github.com/nuxt/content/discussions, since this is for the framework.
